lerna.json

```diff
@@ -1,6 +1,9 @@
 {
   "packages": [
-    "packages/*"
+    "packages/documentation/*",
```

Comment:
I excluded this app from the `lerna bootstrap` process. There are some issues with installing sharp? on linux? I think I need a better understanding of what's going on here before I can fix this. We can just leave it as a todo and handle it in a separate PR. For now this means that the CI won't check anything in the accessibility app. 

```
demo/packages/documentation/node_modules/sharp/build'

  TOUCH Release/obj.target/libvips-cpp.stamp

  CXX(target) Release/obj.target/sharp/src/common.o

../src/common.cc:25:22: fatal error: vips/vips8: No such file or directory

 #include <vips/vips8>

                      ^

compilation terminated.

sharp.target.mk:127: recipe for target 'Release/obj.target/sharp/src/common.o' failed

make: *** [Release/obj.target/sharp/src/common.o] Error 1

make: Leaving directory '/home/jenkins/workspace/ls_tpierce_accessible-react-demo/packages/documentation/node_modules/sharp/build'
```